### PR TITLE
Display captures pieces

### PR DESCRIPTION
**Pull request by joewhale**
* Displayed html of pieces where captured = true
* Still trying to figure out how to write good tests on html being displayed.

### CODE REVIEW

File: app/views/games/show.html.erb [hunk 1]
```diff
@@ -3,6 +3,31 @@
...
 
+    <!-- <div class="captured-pieces">
+      <h1>Captured Pieces</h1>
+
+      <% @game.pieces.where(captured: true).each do |piece| %>
+      <li>
+        <h2>
+          <%= piece.unicode_symbol.html_safe %>
+        </h2>
+      </li>
```
Comment (FsDevNinja) [inline app/views/games/show.html.erb]: we should probably take out this commented out section.

**Comment by lasley** [comment]
@joewhale - My UI people hate me for this, but I typically skip view tests. A view test is an acceptance test, which serves a different purpose than unit & so I like to keep them separate + testing the whole stack.

That said, I'll never deny that more tests are better. In this case we don't really have the full stack either, and no ping-ponging between systems like usually exists. You can use Capybara to accomplish this:
* https://github.com/teamcapybara/capybara#using-capybara-with-rspec

**Comment by joewhale** [comment]
@lasley - That makes sense. I was confused for a while because I wasn't testing the logic, I was just testing the view. I'll try to make sense of the Capybara syntax to try to get a correct view test running. If not, I'll just ask the guys to merge it, so I can work on my other tasks.

**Comment by joewhale** [comment]
@lasley I'm struggling with it because I want to test the view but after I deal with logic like setting some of the pieces' attributes of captured to be 'true'. I am just looping through all the pieces that are captured and displaying their corresponding html code as an <h1> element.

**Comment by lasley** [comment]
This is kind of why I don't like view tests. An acceptance test basically just checks for a bunch of conditions being displayed correctly, which just duplicates your code and creates maintenance burden.

I would instead maybe just assert `$('.piece').length() == known_count`, that the page is displayed with a 200 error code, etc. No need to test everything individually, just use aggregate stuff.

**Pull request merged**